### ISSUE TITLE
Missing package bbros_bridge

### ISSUE THREAD

**Issue by kyordhel** [RobotJustina/JUSTINA#3]
Issue affects branches master and develop

After a fresh install, when catkin_make attempts to build package **_act_pln_** dumps the following error:

```
Could not find a package configuration file provided by "bbros_bridge" with
any of the following names:

    bbros_bridgeConfig.cmake
    bbros_bridge-config.cmake
```

This error prevents the solution from building.

**Comment by mnegretev** [comment]
When you clone the repository, you must use the --recursive option. This happens just because Mister Ra wanted to include its bbros_bridge as a git submodule. PLEASE READ THE GIT TUTORIALS YOU SENT US BY EMAIL. BTW, Yisus says ypur art not following your own "#$"#$"# suggestions.

**Comment by kyordhel** [comment]
Indeed, but that should be also specified within the documentation. 

I was planning to make that remark in the Read.md file on next commit. Still, I open the issue as reminder and for further reference.

**Comment by kyordhel** [comment]
Solved in #5
